src/ugen_xxx.cpp

```diff
@@ -970,258 +972,329 @@ DLL_QUERY lisa_query( Chuck_DL_Query * QUERY )
 {
     Chuck_Env * env = Chuck_Env::instance();
     Chuck_DL_Func * func = NULL;
+    std::string doc;
 
     //---------------------------------------------------------------------
     // init class: LiSa; overloaded class for both LiSaBasic and LiSaMulti
     //              - probably don't need the others anymore....
     // author: Dan Trueman (dan /at/ music.princeton.edu)
     //---------------------------------------------------------------------
 	
-    
+    doc = "LiSa provides basic live sampling functionality.\
+    An internal buffer stores samples chucked to LiSa's input.\
```

Comment:
The spaces at the beginning of the lines are going to add a lot of odd spacing when this is printed out